spec/fixtures/config.yml

```diff
@@ -0,0 +1,20 @@
+# Dummy sample file for testing parsing of configuration file into PushmiPullyu
+
+debug: true
+logdir: tmp/spec/log
+monitor: false
+piddir: tmp/spec/pids
```

Comment:
many of the configurations are used for PMPY and should not be required by this gem.

Reply:
As I said this file is for testing deposit into swift. Just happened to be PMPY config file, but I change it content to be more clear that it is for testing.